Crucial: fix package install. Other minor updates

pcraciunoiu opened the pull request:
**Summary:**
* Fix pip install (really important)
* Add support for upcoming bills (just added to ProPublica API Sept 3rd: https://github.com/propublica/congress-api-docs/issues/8)
* Handle new 404 response format
* Force congress number to always be an integer (fixes default behavior)

For me, the pip install wasn't generating the `congress` directory, so `from congress import Congress` failed. Feel free to fix this a different way, but this was my fix. Works now with:
```
pip install git+git://github.com/UpliftAgency/propublica-congress.git@3ffa3e0#egg=propublica_congress
```

pcraciunoiu committed:
Use sdist and fix package install

eyeseast commented:
Hi. Thanks for the PR. I'm traveling for the holiday weekend, but I'll take a look when I come back. (And I'll see if I can figure out the issue with Travis CI so the tests run.)

pcraciunoiu committed:
Fix 404 and congress version

pcraciunoiu committed:
Add support for upcoming bills

pcraciunoiu commented:
@eyeseast all good. I added a few more small commits, and updated the PR. Thanks for writing this, it saved me some time. I'm happy to contribute as time allows.

eyeseast commented:
OK, ran tests on [another branch](https://travis-ci.org/eyeseast/propublica-congress/builds/272303074) and all passed. I have some thoughts on how to tweak this a little, but this works and is worth merging.

pcraciunoiu commented:
Sweet, thanks! Let me know when this makes it into a release, so I can update the requirements of a project I'm working on :)

eyeseast commented:
Just pushed it up as `v0.3.5`, so you should be good to go.
